src/runtime/runtime.ts

```diff
@@ -196,27 +196,17 @@ export class Runtime {
     return [...this.arcById.values()].find(arc => !!arc.activeRecipe.findParticle(particleId));
   }
 
-  // TODO: This is a temporary method to allow sharing stores with other Arcs.
+  // This is a temporary method to allow sharing stores with other Arcs.
```

Comment:
note to self: rework this change, if https://github.com/PolymerLabs/arcs/pull/4346 lands sooner.